kolibri/plugins/coach/assets/src/vue/exercise-detail-view/index.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+
+  <div id="whole-page">
+    <div id="top-bar">
+      <router-link id="back-btn" :to="pageLink">
+        <mat-svg
+          class="back svg-back"
+          category="navigation"
+          name="arrow_back"
+        />
+        <p class="back">
+          <slot name="text"/>
+        </p>
+      </router-link>
+    </div>
+    <div id="page-body">
+      <slot name="body"/>
+    </div>
+  </div>
+
+</template>
+
+
+<script>
+
+  module.exports = {
+    props: {
+      backPageName: {
+        type: String,
+        required: true,
+      },
+      backPageParams: {
```

Comment:
Might be helpful to instead pass the backPageParams as separate props or add validation here.